viite-UI/src/view/NodeForm.js

```diff
@@ -395,18 +415,22 @@
       $('#nodeStartDate').prop('disabled', isDisabled);
     };
 
-    var nodeChangeHandler = function () {
-      var textIsNonEmpty = $('#nodeName').val() !== "";
-      var nodeTypeIsValid = $('#nodeTypeDropdown :selected').val() !== LinkValues.NodeType.UnknownNodeType.value.toString();
-      var dateIsNonEmpty = $('#nodeStartDate').val() !== "";
-      var saveBtnDisabled = (Environment.name() === 'integration' || Environment.name() === 'production');
-      if (!saveBtnDisabled && textIsNonEmpty && nodeTypeIsValid && dateIsNonEmpty && selectedNode.isDirty()) {
-        $('.btn-edit-node-save').prop('disabled', !saveBtnDisabled);
-      }
+    var disableAutoComplete = function () {
+      $('#nodeName').attr('autocomplete', 'false');
+      $('#nodeStartDate').attr('autocomplete', 'false');
+    };
+
+    var formIsInvalid = function () {
+      //  TODO check if this is still necessary for this branch:
```

Comment:
the Tallenna button should be enabled to all environments after its implementation right?